scripts/globals/abilities/chakra.lua

```diff
@@ -8,48 +8,38 @@
 require("scripts/globals/status")
 -----------------------------------
 
-function onAbilityCheck(player,target,ability)
-    return 0,0
+local ChakraStatusEffects =
+{
+    POISON       = 0, -- Removed by default
+    BLINDNESS    = 0, -- Removed by default
+    PARALYSIS    = 1,
+    DISEASE      = 2,
+    PLAGUE       = 4
+}
+
+function onAbilityCheck(player, target, ability)
+    return 0, 0
 end
 
-function onUseAbility(player,target,ability)
-    local hp = player:getHP()
-    local vit = player:getStat(dsp.mod.VIT)
-    local multi = 2
-    local merits = player:getMerit(dsp.merit.INVIGORATE)
-    local body = player:getEquipID(dsp.slot.BODY)
-    local hand = player:getEquipID(dsp.slot.HANDS)
-
-    if (player:hasStatusEffect(dsp.effect.POISON)) then
-        player:delStatusEffect(dsp.effect.POISON)
-    end
-
-    if (player:hasStatusEffect(dsp.effect.BLINDNESS)) then
-        player:delStatusEffect(dsp.effect.BLINDNESS)
-    end
-
-    if ((body == 12639) or (body == 14474)) then -- Temple Cyclas (+1) equipped
-        if (player:hasStatusEffect(dsp.effect.PARALYSIS)) then
-            player:delStatusEffect(dsp.effect.PARALYSIS)
+function onUseAbility(player, target, ability)
+    local chakraRemoval = player:getMod(dsp.mod.CHAKRA_REMOVAL)
+    if chakraRemoval then
```

Comment:
~= 0, probably

Reply:
Still want it to remove Poison and Blindness.

Reply:
Probably doesn't need to be checked at all, actually.

Reply:
Actually we want it to run if it's 0 for Blindness and Poison removal. So check can probably go away.